sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -28,6 +28,7 @@
      * Instantiates a new query request options.
      */
     public CosmosQueryRequestOptions() {
+        this.queryMetricsEnabled = true;
```

Comment:
Looks like we didn't update the java doc at first place that the default value is `false`. 

@FabianMeiswinkel  - now would be a good time to update the java docs to mention that default value is `true`

Reply:
Fixed in next iteration.